### PR TITLE
approximateGridResolution remarks & note: remove parts of ungeorectified grid, variable cell size grid

### PR DESCRIPTION
**Pull request by tfilppula**
1. Table 20 — S100_DataCoverage parameters, approximateGridResolution, Remarks:

Remove sentence _May be approximate for ungeorectified data (not applicable to this edition of S-102)._

2. Table 20 — S100_DataCoverage parameters, Remove Note: 

_NOTE: If the grid cell size varies over the extent of the grid, an approximated value based on model parameters or production metadata should be used._

### CODE REVIEW

**Review by RohdeBSH — APPROVED** [review]
looks good

**Review by hasel001 — APPROVED** [review]
I agree--it looks good. Thanks!

**Comment by RohdeBSH** [comment]
> Given that we are modifying this part, would it make sense to also add an example in case of unprojected coordinates (geographic)?

Why?
Even if you use a non-projected grid, you have to specify the "approximateGridResolution" in meters.
The attribute type is "real" so it would be something like "7.12345".
In my opinion, the only meaningful modification would be to change "5" to "5.0" to clarify that you can enter decimal numbers.

@tfilppula What's your opinion?

**Comment by giumas** [comment]
If the coordinates are geographic, is it expected to have 2 or 1 approximateGridResolution values? 
Along the N-S axis, how is the suggested approximated value calculated? E.g., it could be the maximum value, the average value. 
In @RohdeBSH's example ("7.12345"), should such a value be truncated or rounded to a significant digit (e.g., "7.123")?
These are 3 aspects that looks unclear/undefined to me in the current formulation of the attribute.

**Comment by tfilppula** [comment]
@giumas @RohdeBSH 
Multiplicity is 1..2 and in the remarks it says: 
```
A single value may be provided when all axes have a common resolution. 
For multiple value provision, use axis order as specified in dataset.
```
So either one value if the ground resolution is the same (approximately) for both axes or two values if they are different. I think the standard is quite clear here.

As the name suggests this is an approximate value - I don't think it is necessary to go for millimetre level. For geographic grids I personally would probably use the mean value for (longitude) resolution but someone else might use something else. If there is a big difference in resolution (north-south), it might be worth considering splitting the dataset to two or more datasets or using UTM projection to minimise scale error. 

Bottom line: I wouldn't change it.

**Comment by giumas** [comment]
@tfilppula, I still believe that an additional example for geographic grids would be useful.

File: sources/3.0.0/sections/12-metadata.adoc [hunk 1]
```diff
@@ -604,18 +604,8 @@ S-102 uses S100_DataCoverage without modification.
 |*Mandatory in S-102* +
 A single value may be provided when all axes have a common resolution. +
 For multiple value provision, use axis order as specified in dataset. +
-May be approximate for ungeorectified data (*not applicable to this edition of S-102*). +
-For example, for 5-metre resolution, the value 5 must be encoded. +
-See note immediately below.
-
+For example, for 5-metre resolution, the value 5 must be encoded.
```
Comment (giumas) [inline sources/3.0.0/sections/12-metadata.adoc:607]: Add an example in case of a geographic grid.

**Review by giumas — CHANGES_REQUESTED** [review]

File: sources/3.0.0/sections/12-metadata.adoc (hunk 1 above)
Comment (tfilppula) [reply sources/3.0.0/sections/12-metadata.adoc:607]: An example of exactly what? Do you have a suggestion?
The note can be brought back, it says:

`NOTE: If the grid cell size varies over the extent of the grid, an approximated value based on model parameters or production metadata should be used.
`

or it can be amended to something like:

`NOTE: If the grid cell size varies over the extent of the grid (when using a geographic grid), an approximated value based on model parameters or production metadata should be used.
`

Comment (giumas) [reply sources/3.0.0/sections/12-metadata.adoc:607]: Restoring the note is also a solution:

`NOTE: If the grid cell size varies over the extent of the grid (i.e., when using a geographic grid), an approximated value should be used.`

Comment (tfilppula) [reply sources/3.0.0/sections/12-metadata.adoc]: I added the following to remarks:
`If the grid cell size varies over the extent of the grid +
(i.e., when using a geographic grid), an approximated value should be used.
`

**Review by giumas — APPROVED** [review]

**Review by RohdeBSH — APPROVED** [review]
